config/detekt/detekt.yml

```diff
@@ -13,6 +13,11 @@ complexity:
     thresholdInClasses: 20
     thresholdInInterfaces: 20
     thresholdInObjects: 20
+  LongParameterList:
+    functionThreshold: 7
+    constructorThreshold: 9
+  NestedBlockDepth:
+    threshold: 6
```

Comment:
Nice try, but no. At least not without some discussion.

Reply:
I was plannig to comment this

Reply:
I'll rebase on develop and give you more details about it

Reply:
`LongParameterList - 8/7 - [Entity] at /home/houcem/Desktop/egm/stellio-context-broker/entity-service/src/main/kotlin/com/egm/stellio/entity/model/Entity.kt:28:13`

`NestedBlockDepth - 5/4 - [updateEntityAttributes] at /home/houcem/Desktop/egm/stellio-context-broker/entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt:567:9`